tap_migrations.json

```diff
@@ -0,0 +1,3 @@
+{
+    "spin": "spinframework/tap"
```

Comment:
I believe this needs to be the full repo path for the new tap
```suggestion
    "spin": "spinframework/homebrew-tap"
```

Reply:
I'm not sure; I was referencing https://github.com/Homebrew/homebrew-core/blob/master/tap_migrations.json and they seem to use `cask`. I think `brew` will auto inject the `homebrew`. @kate-goldenring wdyt?

Reply:
That does seem to be the case! Here is another example https://github.com/Ensembl/homebrew-ensembl/blob/c6979849ed73d59efa32c8a044fa3bddc84f8e00/tap_migrations.json#L4.